File: app/src/main/java/com/kroman/hiitimer/SetupWorkoutActivity.java
package com.kroman.hiitimer;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.databinding.DataBindingUtil;

import com.kroman.hiitimer.databinding.ActivitySetupWorkoutBinding;

/**
 * Created by kromanidis on 13/3/2017.
 */

public class SetupWorkoutActivity extends AppCompatActivity {
    private String workoutName;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivitySetupWorkoutBinding activitySetupWorkoutBinding = DataBindingUtil.setContentView(this, R.layout.activity_setup_workout);
        Toolbar toolbar = activitySetupWorkoutBinding.toolbar;
        toolbar.setTitle(R.string.setup_workout_title);
        setSupportActionBar(toolbar);

        workoutName = activitySetupWorkoutBinding.setWorkoutName.getText().toString();
        activitySetupWorkoutBinding.workoutName.setText(workoutName);

    }
}

File: app/src/main/kotlin/com/kroman/hiitimer/SetupWorkoutActivity.kt
package com.kroman.hiitimer

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import com.kroman.hiitimer.databinding.ActivitySetupWorkoutBinding

class SetupWorkoutActivity : AppCompatActivity() {
    var workoutName: String = ""

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val activitySetupWorkoutBinding: ActivitySetupWorkoutBinding =
            DataBindingUtil.setContentView(this, R.layout.activity_setup_workout)
        val toolbar = activitySetupWorkoutBinding.toolbar
        toolbar.setTitle(R.string.setup_workout_title)
        setSupportActionBar(toolbar)

        workoutName = activitySetupWorkoutBinding.setWorkoutName.text.toString()
        activitySetupWorkoutBinding.workoutName.text = workoutName
    }
}